### PR TITLE
Add package name to health connect sensors, additional sensor tweaks

### PR DESCRIPTION
**Pull request by dshokouhi**
## Summary

- update weight icon to be more correct
- Add state class total increasing for total calories as the sensor resets at midnight and accumulates until then
- Add package as an attribute so users can understand from where the data came from

## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#

### CODE REVIEW

File: app/src/full/java/io/homeassistant/companion/android/sensors/HealthConnectSensorManager.kt [hunk 1]
```diff
@@ -104,13 +105,16 @@ class HealthConnectSensorManager : SensorManager {
...
             onSensorUpdated(
                 context,
                 totalCaloriesBurned,
-                BigDecimal(it.inKilocalories).setScale(2, RoundingMode.HALF_EVEN),
+                BigDecimal(energy.inKilocalories).setScale(2, RoundingMode.HALF_EVEN),
                 totalCaloriesBurned.statelessIcon,
-                attributes = mapOf("endTime" to LocalDateTime.of(LocalDate.now(), LocalTime.now()).toInstant(ZoneOffset.UTC))
+                attributes = mapOf(
+                    "endTime" to LocalDateTime.of(LocalDate.now(), LocalTime.now()).toInstant(ZoneOffset.UTC),
+                    "package" to totalCaloriesBurnedRequest.dataOrigins.map { it.packageName }
```
Comment (jpelgrom) [inline app/src/full/java/io/homeassistant/companion/android/sensors/HealthConnectSensorManager.kt:116]: This returns a `Set` so you're creating a list here, is that intentional? If so maybe it is worth adding a `s` to the attribute name to differentiate it.

**Review by jpelgrom — COMMENTED** [review]
I'm not sure the attribute should be `package` or if `source` makes more sense for less technical users 🤔

Other changes look good to me

**Comment by jpelgrom** [comment]
Also, was Health Connect approved? Or is this just cleaning it up in case it is like the other cleanups we've done?

File: app/src/full/java/io/homeassistant/companion/android/sensors/HealthConnectSensorManager.kt (hunk 1 above)
Comment (dshokouhi) [reply app/src/full/java/io/homeassistant/companion/android/sensors/HealthConnectSensorManager.kt:116]: I made it into a list as I was unsure a `Set` would translate properly in teh attribute so wanted to use something I knew worked :) and I agree adding a `s` here makes sense.

**Comment by dshokouhi** [comment]
> I'm not sure the attribute should be `package` or if `source` makes more sense for less technical users 🤔

hmm `source` feels more correct I think I will update

**Review by jpelgrom — APPROVED** [review]

**Pull request merged**